substrate/frame/mixnet/src/lib.rs

```diff
@@ -368,12 +360,12 @@ impl<T: Config> Pallet<T> {
 			.saturating_sub(CurrentSessionStartBlock::<T>::get());
 		let Some(block_in_phase) = block_in_phase.checked_sub(&T::NumCoverToCurrentBlocks::get())
 		else {
-			return SessionPhase::CoverToCurrent
+			return SessionPhase::CoverToCurrent;
 		};
 		let Some(block_in_phase) =
 			block_in_phase.checked_sub(&T::NumRequestsToCurrentBlocks::get())
 		else {
-			return SessionPhase::RequestsToCurrent
+			return SessionPhase::RequestsToCurrent;
```

Comment:
This is likely because your formatter is different than what we use in CI. 

You might want to ask in the CI helpdepsk to once and for all document where one should find the proper rust versions for all operations (test, fmt), if you have a hard time finding it.

Reply:
I asked in the CI helpdesk and they pointed me to `.github/env` which only has this configuration:

```env
IMAGE="docker.io/paritytech/ci-unified:bullseye-1.81.0-2024-11-19-v202411281558"
```

I checked the `fmt` workflow as well and that has this configuration - 

```yml
- name: Cargo fmt
   id: required
   run: cargo +nightly fmt --all -- --check
```

I am using the nightly toolchain as well for `fmt` and given that the GitHub workflow is succeeding, I think this formatting is correct.

Here is my configuration:

```bash
rustc --version
rustc 1.81.0 (eeb90cda1 2024-09-04)

cargo --version
cargo 1.81.0 (2dbb1af80 2024-08-20)

rustc +nightly --version
rustc 1.85.0-nightly (c26db435b 2024-12-15)

cargo +nightly --version
cargo 1.85.0-nightly (769f622e1 2024-12-14)
```